common/source/docs/common-yaapu-gcs.rst

```diff
@@ -0,0 +1,42 @@
+.. _common-yaapu-gcs:
+
+====================================================
+Yaapu FrSky GCS using FrSky Bi-Directional Telemetry
+====================================================
+
+ArduPilot now has the capability of not only displaying downlink telemetry data from the autopilot via FrSky telemetry (see :ref:`common-frsky-telemetry` )for display on an OpenTX compatible transmitter, but also the ability to request and change parameters and send a few commands to the autopilot.
+
+Taranis Display Example
+-----------------------
+
+.. image:: ../../../images/X9Dexample2.png
+    :width: 400px
+
+QX7 Display Example
+-------------------
+
+.. image:: ../../../images/QX7example1.png
+
+
+Horus/T16 Example
+-----------------
+
+.. image:: ../../../images/Horus1.png
+
+
+
+Overview
+========
+
+In order to use this ability, you will need to setup the Yaapu LUAGCS script on your OpenTX transmitter. This script can be run con-currently with the Yaapu Telemetry script, but it is not required. Of course, the transmitter's display will only be used by one or the other at a time, but if the Telemetry script is running, important messages,warnings, and alerts will still be displayed while using the LUAGCS.
```

Comment:
When the GCS script is running all other lua scripts are put on hold by OpenTX.
On the Horus the Lua GCS has telemetry info and status text messages and history (no warnings or sound alerts) while on the Taranis only battery voltage is displayed and there's no status text message history.

Reply:
thanks...will correct...otherwise,okay?